### PR TITLE
Add LOWER love letter

### PR DESCRIPTION
**Pull request by johnblust**
Adds canonical LOWER love letter to DevBlog

### CODE REVIEW

File: website/blog/2022-06-30-lower-sql-function.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+In the real world of human imperfection, opinions, and error, string values are likely to take inconsistent capitalization across different data sources (or even within the same data source). There’s always a little lack of rhyme or reason for why some values are passed as upper or lowercase, and it’s not worth the headache to unpack that.
+
+So how do you create uniformity for string values that you collect across all your data sources? The LOWER function!
+
+<!--truncate-->
+
+Using the LOWER function on a string value will return the input as an all lowercase string. It’s an effective way to create consistent capitalization for string values across your data.
+
+> **What is a SQL function?** 
+> At a high level, a function takes an input (or multiple inputs) and returns a manipulation of those inputs. Some common SQL functions are COALESCE, EXTRACT, and DATEDIFF. For example, the `COALESCE` function takes a group of values and returns the first non-null value from that group.
```
Comment (KiraFuruichi) [inline website/blog/2022-06-30-lower-sql-function.md]:
```suggestion
>
> At a high level, a function takes an input (or multiple inputs) and returns a manipulation of those inputs. Some common SQL functions are COALESCE, EXTRACT, and DATEDIFF. For example, the COALESCE function takes a group of values and returns the first non-null value from that group.
```

Can we mark somewhere to link to COALESCE, EXTRACT and DATEDIFF pages?

File: website/blog/2022-06-30-lower-sql-function.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+## How to use the LOWER function
+
+The syntax for using the LOWER function looks like the following:
+
+```sql
+lower(‘<string_value>’)
```
Comment (KiraFuruichi) [inline website/blog/2022-06-30-lower-sql-function.md]:
```suggestion
lower('<string_value>')
```

File: website/blog/2022-06-30-lower-sql-function.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+Executing this command in a SELECT statement will return the lowercase version of the input string. You may additionally use the LOWER function in WHERE clauses and joins.
+
+### Data warehouse support for the LOWER function
+
+Google BigQuery, Amazon Redshift, Snowflake, Postgres, and Databricks all support the LOWER function. In addition, the syntax to use LOWER is the same across all of them.
+
+## LOWER SQL function example
+
+Let’s take this to an actual example! Below, you’ll see the first three rows from the `customers` <Term id="table" /> in the jaffle_shop, a simple dataset and dbt project, that has three columns: `customer_id`, `first_name`, and `last_name`.
```
Comment (KiraFuruichi) [inline website/blog/2022-06-30-lower-sql-function.md]:
```suggestion
Let’s take this to an actual example! Below, you’ll see the first three rows from the `customers` <Term id="table" /> in the [jaffle_shop](https://github.com/dbt-labs/jaffle_shop), a simple dataset and dbt project, that has three columns: `customer_id`, `first_name`, and `last_name`.
```

File: website/blog/2022-06-30-lower-sql-function.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+| 3               | Kathleen       | P.            |
+
+You can lower the first name and last name of the `customers` model using the following code:
+
+```
+select 
+	customer_id,
+	lower(first_name) as first_name,
+	lower(last_name) as last_name
+from {{ ref(‘customers’) }}
```
Comment (KiraFuruichi) [inline website/blog/2022-06-30-lower-sql-function.md]:
```suggestion
from {{ ref('customers') }}
```

File: website/blog/2022-06-30-lower-sql-function.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+```sql
+lower(‘<string_value>’)
+```
+
+Executing this command in a SELECT statement will return the lowercase version of the input string. You may additionally use the LOWER function in WHERE clauses and joins.
+
+### Data warehouse support for the LOWER function
+
+Google BigQuery, Amazon Redshift, Snowflake, Postgres, and Databricks all support the LOWER function. In addition, the syntax to use LOWER is the same across all of them.
```
Comment (KiraFuruichi) [inline website/blog/2022-06-30-lower-sql-function.md]:
```suggestion
[Google BigQuery](https://cloud.google.com/bigquery/docs/reference/standard-sql/string_functions#lower), [Amazon Redshift](https://docs.aws.amazon.com/redshift/latest/dg/r_LOWER.html), [Snowflake](https://docs.snowflake.com/en/sql-reference/functions/lower.html), [Postgres](https://www.postgresqltutorial.com/postgresql-string-functions/postgresql-letter-case-functions/), and [Databricks](https://docs.databricks.com/sql/language-manual/functions/lower.html) all support the LOWER function. In addition, the syntax to use LOWER is the same across all of them.
```

File: website/blog/2022-06-30-lower-sql-function.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+| **customer_id** | **first_name** | **last_name** |
+| --------------- | -------------- | ------------- |
+| 1               | michael        | p.            |
+| 2               | shawn          | m.            |
+| 3               | kathleen       | p.            |
+
+Now, all characters in the `first_name` and `last_name` columns are lowercase.
+
+> **Note:**
+> Changing all string columns to lowercase to create uniformity across data sources typically happens in our dbt project’s staging models. There are a few reasons for that: data cleanup and standardization, such as aliasing, casting, and lowercasing, should ideally happen in staging models to create downstream uniformity. It’s also more performant in downstream models that join on string values to join on strings that are of all the same casing versus having to join and perform lowercasing at the same time.
```
Comment (KiraFuruichi) [inline website/blog/2022-06-30-lower-sql-function.md]:
```suggestion
> Changing all string columns to lowercase to create uniformity across data sources typically happens in our dbt project’s [staging models](https://docs.getdbt.com/guides/best-practices/how-we-structure/2-staging). There are a few reasons for that: data cleanup and standardization, such as aliasing, casting, and lowercasing, should ideally happen in staging models to create downstream uniformity. It’s also more performant in downstream models that join on string values to join on strings that are of all the same casing versus having to join and perform lowercasing at the same time.
```

**Review by KiraFuruichi — CHANGES_REQUESTED** [review]
Some minor edits, mostly links missing!

File: website/blog/2022-06-30-lower-sql-function.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+| **customer_id** | **first_name** | **last_name** |
+| --------------- | -------------- | ------------- |
+| 1               | Michael        | P.            |
+| 2               | Shawn          | M.            |
+| 3               | Kathleen       | P.            |
+
+You can lower the first name and last name of the `customers` model using the following code:
+
+```
```
Comment (KiraFuruichi) [inline website/blog/2022-06-30-lower-sql-function.md]:
```suggestion
```sql
```

I was wondering why the colors weren't displaying correctly!

**Comment by johnblust** [comment]
Thanks for the review Kira. I added the links. I'll do this for the rest as well since we'll be merging them at the same time.

**Review by KiraFuruichi — APPROVED** [review]
dope dope dope

**Pull request merged**